Contact pairing

hetrox8 opened the pull request:
This is branch constitutes of contact form where visitors can leave there message. The language used to achieve this is html and css.

V-Blaze commented on index.html:
```diff
@@ -345,6 +346,7 @@ <h6 class="h-contact"> contact me</h6>
 
         </form>
```
- [ ] You did great here, :+1: Your form functions as expected, but there are some styling issues you need to fix before moving to the next milestone.

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [ ] As part of the requirement is to replicate the given design as much as you can, Looking at the screenshot from the Figma design below, you are missing some background images, I'll suggest you get all the images and update your UI accordingly.
- [ ] Also the position of your top background image is different from the given design, the background image is behind the text and it makes it hard to see and read the text, I'll suggest you update this image position too. :+1: :smile: 

- - Your design:
![Selection_059](https://github.com/hetrox8/portfolio1/assets/107632780/0602e621-6666-4b6f-86ca-ea525104646a)

- - Figma design:
![Selection_058](https://github.com/hetrox8/portfolio1/assets/107632780/49e24fea-a5a6-4b1c-997a-0dfe73fa4120)

V-Blaze commented on index.html:
```diff
@@ -345,6 +346,7 @@ <h6 class="h-contact"> contact me</h6>
 
         </form>
       </div>
+    </div>
```
- [ ] You did great using the button tag here. :+1: But I think there are still some styling issues you need to update here. 
- [ ] Currently, the position of your button is different from the one in the given design, I'll suggest you update your CSS as replicating the given design is one of the requirements for this milestone. :+1: 

- - YOUR CURRENT DESIGN:
![Screenshot from 2023-06-07 10-24-27](https://github.com/hetrox8/portfolio1/assets/107632780/14f1f7f1-06c9-4916-8271-27086892f314)

- - FIGMA DESIGN:
![Screenshot from 2023-06-07 10-24-12](https://github.com/hetrox8/portfolio1/assets/107632780/97a0707d-abad-4a20-ac4f-c8dfaf0d693a)

- [ ] Also your buttons do not have the interactive properties implemented, You are meant to implement the `hover and pressed` state of your button with the correct background color and text color. I'll suggest you update this too. :+1: :100: 

- - HOVER AND PRESSED STATE FROM DESIGN:
![Screenshot from 2023-06-07 09-16-03](https://github.com/HossainAraf/Portfolio-Mobile-First/assets/107632780/15649450-132c-4ef2-af7e-bba644996e85)

V-Blaze reviewed (changes requested):
## Hi @hetrox8, @NalianAchei2020  

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

![giphy](https://user-images.githubusercontent.com/107632780/229314554-3700de13-2c00-46d5-bc77-46ad09af7467.gif)


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Correct Github flow✔
-  Form submission works great👋 
-  Readme looks professional ✔

 ## Required Changes ♻️

## Optional suggestions
- [OPTIONAL] Your commit messages are meaningful, but following the best practice I think you can improve them by doing these, 
-  - you are meant to capitalize the first letter of the first word in your commit messages, I'll suggest you put this into practice in your upcoming projects. :+1: :smile: 
- - Also instead of using generic comments like `make changes` it would be better of you actually say what changes were made. eg `Add form post action to Formspree service.`

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

hetrox8 commented:
1. Added background image
2. Added background position for  top part
3. Changed button position to match  the required position to match figma desing
4. Included button hover effect

OyePriscilla commented on styles.css:
```diff
@@ -490,10 +488,20 @@ textarea::placeholder {
   margin-top: 5%;
 }
 
+.cont-btn:hover {
+  background-color: rebeccapurple;
+  color: #fff;
+  cursor: pointer;
+}
+
+.cont-btn:active {
+  background-color: #fff;
+}
+
```
- [ ] Kindy ensure that you design these pseudo classes according to the given template. `#6070FF` and `#2230D2`
 are the background-colors for hover and active states respectively. 👍 

![DSC_8108](https://github.com/hetrox8/portfolio1/assets/103386183/609aa4f9-5d35-4c16-bffa-f647b0a22efc)

- [ ] Kindly ensure that the button element is aligned properly with other input element according to the design. Implementing this will give your UI a nice interface. 👍

OyePriscilla reviewed (changes requested):
Hi @hetrox8 👍 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![so-close-so-close-seth-meyers](https://user-images.githubusercontent.com/103386183/196425724-8ca5825c-b02a-4983-b976-4bd0a2f70f9a.gif)

 ## Highlights
Implemented background-image with CSS ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

OyePriscilla reviewed (changes requested):
Hi @hetrox8 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![200](https://user-images.githubusercontent.com/103386183/196424328-985f6e98-f7c6-4cf9-bafe-158436f70560.gif)

## Highlights
Implemented background-image with CSS ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

christianbiring1 commented:
Hey @hetrox8 , Your project is approved now. seems like it was a little mistake from the previous reviewer, YOu can go ahead and submit now.

Happy coding :clap:

OyePriscilla commented:
Hi @hetrox8, I'm so sorry for the confusion; I meant to require changes in the PR as well. ℹ️
